packages/react-liveness/src/components/FaceLivenessDetector/service/utils/liveness.ts

```diff
@@ -326,11 +326,28 @@ function getPupilDistanceAndFaceHeight(face: Face) {
   return { pupilDistance, faceHeight };
 }
 
-export function generateBboxFromLandmarks(
-  face: Face,
-  oval: LivenessOvalDetails,
-  frameHeight: number
-): BoundingBox {
+export function generateBboxFromLandmarks({
+  sessionInformation,
+  face,
+  oval,
+  frameHeight,
+}: {
+  sessionInformation: SessionInformation;
+  face: Face;
+  oval: LivenessOvalDetails;
+  frameHeight: number;
+}): BoundingBox {
+  const challengeConfig =
+    sessionInformation?.Challenge?.FaceMovementAndLightChallenge
+      ?.ChallengeConfig;
+
+  if (!challengeConfig || !challengeConfig.OvalHeightWidthRatio) {
```

Comment:
Do we need to perform this check here if we already have done the same check [here](https://github.com/aws-amplify/amplify-ui/pull/5254/files#diff-cca8afb5864b273535f7e509efb269bdfc349fa9f544501a16443ce86a7e9680R1012)?

Reply:
removed!